--- ./src/ros2/rviz/rviz_default_plugins/CMakeLists.txt 2023-08-20 09:41:18
+++ src/ros2/rviz/rviz_default_plugins/CMakeLists.txt        2023-08-20 08:08:36
@@ -57,7 +57,8 @@
 find_package(rviz_common REQUIRED)
 find_package(rviz_rendering REQUIRED)
 find_package(rviz_ogre_vendor REQUIRED)
 
+include_directories(/opt/homebrew/Cellar/qt@5/5.15.11/include)
 find_package(Qt5 REQUIRED COMPONENTS Widgets Test)
 
 find_package(geometry_msgs REQUIRED)
@@ -237,7 +238,8 @@
 target_include_directories(rviz_default_plugins PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/${PROJECT_NAME}>
   ${Qt5Widgets_INCLUDE_DIRS}
+  "/opt/homebrew/Cellar/orocos-kdl/1.5.1/include/"
 )
 
 target_link_libraries(rviz_default_plugins PUBLIC
