--- src/ros-visualization/interactive_markers/CMakeLists.txt	2023-06-09 07:56:04
+++ src/ros-visualization/interactive_markers/CMakeLists.txt	2023-06-12 21:52:16
@@ -19,6 +19,8 @@
 find_package(tf2_geometry_msgs REQUIRED)
 find_package(visualization_msgs REQUIRED)
 
+include_directories(${orocos_kdl_INCLUDE_DIRS})
+
 ament_python_install_package(${PROJECT_NAME})
 
 add_library(${PROJECT_NAME} SHARED
