--- src/ros2/rosbag2/rosbag2_transport/src/rosbag2_transport/play_options.cpp
+++ src/ros2/rosbag2/rosbag2_transport/src/rosbag2_transport/play_options.cpp
@@ -62,7 +62,11 @@ Node convert<rosbag2_transport::PlayOptions>::encode(
 bool convert<rosbag2_transport::PlayOptions>::decode(
   const Node & node, rosbag2_transport::PlayOptions & play_options)
 {
-  optional_assign<uint64_t>(node, "read_ahead_queue_size", play_options.read_ahead_queue_size);
+  //  optional_assign<uint64_t>(node, "read_ahead_queue_size", play_options.read_ahead_queue_size);
+  uint64_t temp = static_cast<uint64_t>(play_options.read_ahead_queue_size);
+  optional_assign<uint64_t>(node, "read_ahead_queue_size", temp);
+  play_options.read_ahead_queue_size = static_cast<size_t>(temp);
+
   optional_assign<std::string>(node, "node_prefix", play_options.node_prefix);
   optional_assign<float>(node, "rate", play_options.rate);
   optional_assign<std::vector<std::string>>(
