--- src/ros-perception/point_cloud_transport/point_cloud_transport/include/point_cloud_transport/point_cloud_transport.hpp
+++ src/ros-perception/point_cloud_transport/point_cloud_transport/include/point_cloud_transport/point_cloud_transport.hpp
@@ -269,7 +269,7 @@ public:
     return subscribe(
       base_topic, custom_qos, std::bind(
         fp,
-        obj.get(), std::placeholders::_1), VoidPtr(), transport_hints);
+        obj, std::placeholders::_1), VoidPtr(), transport_hints);
   }

   template<class T>
@@ -282,7 +282,7 @@ public:
     return subscribe(
       base_topic, queue_size, std::bind(
         fp,
-        obj.get(), std::placeholders::_1), VoidPtr(), transport_hints);
+        obj, std::placeholders::_1), VoidPtr(), transport_hints);
   }

   //! Subscribe to a point cloud topic, version for class member function with shared_ptr.
