--- ./src/ros2/rviz/rviz_common/CMakeLists.txt  2023-08-20 09:41:18
+++ src/ros2/rviz/rviz_common/CMakeLists.txt 2023-08-20 07:43:49
@@ -31,6 +31,7 @@
 # do find_package(rviz_ogre_vendor) first to make sure the custom OGRE is found
 find_package(rviz_ogre_vendor REQUIRED)
 
+include_directories(/opt/homebrew/Cellar/qt@5/5.15.11/include)
 find_package(Qt5 REQUIRED COMPONENTS Widgets)
 
 find_package(geometry_msgs REQUIRED)
