--- ./src/ros2/rviz/rviz_visual_testing_framework/CMakeLists.txt        2023-08-20 09:41:18
+++ src/ros2/rviz/rviz_visual_testing_framework/CMakeLists.txt       2023-08-20 08:18:33
@@ -23,6 +23,9 @@
   add_compile_options(-Wall -Wextra -Wpedantic -Wnon-virtual-dtor -Woverloaded-virtual)
 endif()
 
+
+include_directories(/opt/homebrew/Cellar/qt@5/5.15.12/include)
+
 find_package(ament_cmake REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(Qt5 REQUIRED COMPONENTS Widgets Test)
 